packages/api/tests/test-ems-report.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
```

Comment:
I think we should also do some testing of the format of the records.

Reply:
+1

Reply:
added format checking